src/test/scala/is/hail/io/ImportMatrixSuite.scala

```diff
@@ -6,62 +6,118 @@ import is.hail.check.Gen
 import is.hail.check.Prop.forAll
```

Comment:
General comment: I wonder if these tests would look more natural in python? Currently they're a bit wordy and it's hard to parse out exactly what's happening.